### ISSUE TITLE
yay penguins-eggs refuse to install

### ISSUE THREAD

**Issue by pieroproietti** [pieroproietti/penguins-eggs-arch#3]
yay penguins-eggs refuse to install becouse don't find mkcpio-archiso on manjaro.

This Is Just a note for Mark, tomorrow i Will complete the issue

**Comment by yochananmarqos** [comment]
We blacklist some Arch tools like `mkinitcpio-archiso` because Manjaro is not Arch; Arch is not Manjaro. We have our own tools as I'm fairly certain you're already aware of:

```
❯ pacman -Ss manjaro-tools
extra/manjaro-tools-base-git r3004.dc48e4e-2 (manjaro-tools)
    Development tools for Manjaro Linux (base tools)
extra/manjaro-tools-iso-git r3004.dc48e4e-2 (manjaro-tools)
    Development tools for Manjaro Linux (ISO tools)
extra/manjaro-tools-pkg-git r3004.dc48e4e-2 (manjaro-tools)
    Development tools for Manjaro Linux (packaging tools)
extra/manjaro-tools-yaml-git r3004.dc48e4e-2 (manjaro-tools)
    Development tools for Manjaro Linux (yaml tools)
```

@Ste74 As I already mentioned, #2 makes no sense as `mkinitcpio-archiso` and `manjaro-tools-iso` are not interchangable.

**Comment by Ste74** [comment]
Yep,  issue Is that: with makepkg recognition of OS work fine but with any aur wrapper no so i suggest Piero a dirty solution: remove from depends mkinitcpio-archiso and put a statment into eggs code. At First run of eggs It check the OS and then install via Pacman One or other follow the host system. To leave the system clean also use a .install script for post-remove message for invite the user to uninstall One or other. This Is very dirty but only One pkgbuild Is provided and maintained.
Otherwise if you are Good with Muflone changes we can reimport in our repo 🤔

**Comment by pieroproietti** [comment]
I had gotten up this morning wanting to write the issue decently, you fooled me though.... :-D

I, but I think also other developers, could use that once the program creation is finished and checked that it works the work is done.

The success of Arch, is also this: to leave - as much as possible - the sources as they are by coating them with a light PKGBUILD.

Being able to use an ```if Manjaro```, ```else``` then 
```depends+=('manjaro-tools-iso')``` would help not a little, at least from my point of view.

Unfortunately, if you have the need of blacklist some packages, I see no other way but to keep versions separate: one for Arch, one for Manjaro, and so on. 

This makes it artificially complex to maintain a program for the entire Arch family compared to Debian/Devuan/Ubuntu and derivatives.

I hope I have not blasphemed in some way, if so, it is due to my ignorance about the Arch world where I have much to learn.

**Comment by pieroproietti** [comment]
After many tempts, I have removed manjaro-iso-tools from the PKGBUILD named penguins-eggs in AUR, thus making it exclusively Arch. 

At this point, I have also created a new PKGBUILD for Manjaro currently in my repo [penguins-eggs-manjaro](https://github.com/pieroproietti/penguins-eggs-manjaro).

I hope that the current PKGBUILD  penguins-eggs-manjaro, can be accepted in community on this distribution.

@yochananmarqos @Ste74

**Comment by pieroproietti** [comment]
Starting with v9.4.3 penguins-eggs is included in the [Manjaro community repo](https://gitlab.manjaro.org/packages/community/penguins-eggs)!
